api/data/seattle_db.go

```diff
@@ -97,18 +97,18 @@ func (c *Client) SeattleOfficerMetadata() *DepartmentMetadata {
 		},
 		LastAvailableRosterDate: date.Format("2006-01-02"),
 		Name:                    "Seattle PD",
-		ID:                      "spd",
+		ID:                      "seattle-wa",
```

Comment:
I'm pretty sure we'll also need to update 1-312 with this new ID here: https://github.com/OrcaCollective/1-312-hows-my-driving/blob/main/src/app.py#L106